src/community/jwt-headers/jwt-headers-util/src/main/java/org/geoserver/security/jwtheaders/JwtConfiguration.java

```diff
@@ -1,8 +1,7 @@
 package org.geoserver.security.jwtheaders;
 
 import java.io.Serializable;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
GeoServer policies want to avoid importing the whole package but single classes instead.